//#include <cstdio>
//#include <iostream>
import test;


int main() {
  //try{
    //printf("printfmain\n");
    //std::cout<<"coutmain"<<std::endl;
    /*}catch(std::exception e){
      printf("%s\n",e.what());
    }*/
  testfn();
}
